embassy-rp/src/uart/mod.rs

```diff
@@ -146,6 +153,29 @@ impl<'d, T: Instance, M: Mode> UartTx<'d, T, M> {
         unsafe { while !r.uartfr().read().txfe() {} }
         Ok(())
     }
+
+    pub fn busy(&self) -> bool {
+        unsafe { T::regs().uartfr().read().busy() }
+    }
+
+    /// Assert a break condition after waiting for the transmit buffers to empty,
+    /// or release a previously asserted break condition. This condition must be
+    /// asserted for at least two frame times to be effective, otherwise it may
+    /// be received as a normal character.
+    pub fn set_break(&mut self, brk: bool) {
```

Comment:
why not handle the timing ourselves, so the user does just `uart.send_break()` and doesn't have to worry about the timing?

Reply:
just use `embassy-time`, so no need to mess with sysclk.